thespian/system/transport/TCPTransport.py

```diff
@@ -639,8 +638,8 @@ def _next_XMIT_1(self, intent):
                 return self._nextTransmitStep(intent)
             # If there is an active or pending Intent for this target,
             # just queue this one (by returning True)
-            if [T for T in self._transmitIntents.values()
-                if T.targetAddr == intent.targetAddr and hasattr(T, 'socket')]:
+            if any(T for T in self._transmitIntents.values()
+                if T.targetAddr == intent.targetAddr and hasattr(T, 'socket')):
```

Comment:
Theoretically a performance enhancement, I think it's also a little clearer to use `any()` here.